yarn.lock

```diff
@@ -5185,9 +5185,9 @@ lodash.padstart@^4.1.0:
   resolved "https://registry.yarnpkg.com/lodash.padstart/-/lodash.padstart-4.6.1.tgz#d2e3eebff0d9d39ad50f5cbd1b52a7bce6bb611b"
   integrity sha1-0uPuv/DZ05rVD1y9G1KnvOa7YRs=
 
-lodash.sortby@^4.7.0:
+lodash.sortKey@^4.7.0:
```

Comment:
Don't edit `yarn.lock` manually, this is the file which determines which packages are installed and is auto generated. https://yarnpkg.com/lang/en/docs/yarn-lock/ -- this removes the dependency and replaces it with a non-existent one